src/collections/blog/2022/2022-20-06-managing-multiple-clusters-with-meshery/index.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Managing Multiple clusters with Meshery
+subtitle: Now Meshery is supporting Kubernetes multi-cluster features
+date: 2022-06-20 08:00:00 -0630
+author: Ashish Tiwari
+description: "Philosophy and approach of managing multiple clusters with Meshery"
+thumbnail: ./thumbnail.png
+category: "Meshery"
+tags:
+ - Meshery
+ - Multi-cluster
+ - Kubernetes
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+
+<BlogWrapper>
+
+# Managing multiple clusters with Meshery
+<p align="center">
+  <img src="./thumbnail.png" />
```

Comment:
Can you try importing all images on top and then referencing them here?

Reply:
@debo19 Thank you that worked!